tests/unit/panther_analysis_tool/test_main.py

```diff
@@ -38,17 +39,19 @@ def test_handle_wrong_key_error(self):
         expected_output = '{} not in list of valid keys: {}'
         # test successful regex match and correct error returned
         test_str = "Wrong key 'DisplaName' in {'DisplaName':'one','Enabled':true, 'Filename':'sample'}"
-        exc = Exception(test_str)
+        exc = SchemaWrongKeyError(test_str)
         err = pat.handle_wrong_key_error(exc, sample_keys)
         assert_equal(str(err), expected_output.format("'DisplaName'", sample_keys))
         # test failing regex match
         test_str = "Will not match"
-        exc = Exception(test_str)
+        exc = SchemaWrongKeyError(test_str)
         err = pat.handle_wrong_key_error(exc, sample_keys)
         assert_equal(str(err),  expected_output.format("UNKNOWN_KEY", sample_keys))
 
     def test_load_policy_specs_from_folder(self):
         args = pat.setup_parser().parse_args('test --path tests/fixtures'.split())
+        args.filter = None
```

Comment:
Curious why this is required?

Reply:
I added the `dest` parameter to the `subparser.add_argument` function that maps the user input to `filter_list`
This was done for type resolving since we were remapping `filter` from a list to a dict which complicated the typing

Reply:
The reason it is required is because `args.filter` is not instantiated by `parse_args` now, it is now instantiated by the line `args.filter = parse_filter(args.filter_list)`